tests/units/main/global_configuration_test.py

```diff
@@ -28,6 +28,9 @@
 
 class GlobalConfiguration_TestCase(unittest.TestCase):
 
+    def setUp(self):
+        App.initialize()
+
     def tearDown(self):
         App.initialize()
```

Comment:
Do you need to initialize it again in teardown? I'd expect it to be something like cleanup, free, destroy, not the same.

Reply:
Thanks, that makes sense.